File: src/main/java/BoxTester.java
import java.util.Scanner;

public class BoxTester {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Please enter object's length");
        float length = input.nextFloat();
        System.out.println("Please enter object's width");
        float width = input.nextFloat();
        System.out.println("Please enter object's height");
        int height = input.nextInt();
        Box3 box3 = new Box3();
        Box5 box5 = new Box5();
        if (box3.validate(length,width,height)) {
            System.out.println("你的包裹適合Box3的尺寸");
        } else if (box5.validate(length,width,height)){
            System.out.println("你的包裹適合Box5的尺寸");
        } else {
            System.out.println("沒有合適尺寸的Box");
        }

    }
}

File: src/main/kotlin/BoxTester.kt
import java.util.*
import com.hanson1.Box3
import com.hanson1.Box5

fun main(args: Array<String>) {
    var input = Scanner(System.`in`)
    println("Please enter object's1 length")
    val length = input.nextFloat()
    println("Please enter object's1 width")
    val width = input.nextFloat()
    println("Please enter object's1 height")
    val height = input.nextInt()
    val box3 = Box3()
    val box5 = Box5()
    if (box3.validate(length, width, height)) {
        println("你的包裹適合Box3的尺寸")
    } else if (box5.validate(length, width, height)) {
        println("你的包裹適合Box5的尺寸")
    } else {
        println("沒有合適尺寸的Box")
    }
}